### PR TITLE
Add the an output format for Mosaic

### PR DESCRIPTION
**Pull request by mfairchild365**
Mosaic apparently requires its own custom xml format rather than using an existing standard.

### CODE REVIEW

File: www/templates/xml/MediaList.tpl.php [hunk 1]
```diff
@@ -4,4 +4,3 @@
         echo $savvy->render($media);
     }
 }
-?>
```
Comment (kabel) [inline www/templates/xml/MediaList.tpl.php]: Oh no. Got rid of the newline!!!!

File: www/templates/xml/DefaultHomepage.tpl.php [hunk 2]
```diff
@@ -1,2 +1,2 @@
 <?php
-echo $savvy->render($context->top_media);
\ No newline at end of file
+echo $savvy->render($context->getTopMedia());
```
Comment (kabel) [inline www/templates/xml/DefaultHomepage.tpl.php]: Love the getter; hate the missing newline.

File: www/templates/xml/MediaList.tpl.php (hunk 1 above)
Comment (mfairchild365) [inline www/templates/xml/MediaList.tpl.php]: Naw it is still there.

File: www/templates/xml/DefaultHomepage.tpl.php (hunk 2 above)
Comment (mfairchild365) [inline www/templates/xml/DefaultHomepage.tpl.php]: Doh'

**Pull request merged**